Tests/js/views/assets/ez-usermenuview-tests.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview-tests', function (Y) {
+    var renderTest, eventTest,
+        CLASS_HIDDEN = 'is-user-menu-hidden';
+
+    renderTest = new Y.Test.Case({
+        name: "eZ User Menu render test",
+
+        setUp: function () {
+            this.userMenuItemFireEventView = new Y.View();
+            this.view = new Y.eZ.UserMenuView({
+                items: [this.userMenuItemFireEventView]
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should render the user menu items": function () {
+            var container = this.view.render().get('container');
+
+            Y.Assert.areSame(
+                1,
+                container.get('children').size(),
+                'The user menu should have one child'
+            );
+        },
+    });
+
+    eventTest = new Y.Test.Case({
+        name: "eZ User Menu event test",
+
+        setUp: function () {
+            this.userMenuItemFireEventView = new Y.View();
+            this.view = new Y.eZ.UserMenuView({
+                items: [this.userMenuItemFireEventView],
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should show the user menu": function () {
+            var view = this.view,
+                container = view.render().get('container');
+
+            view._set('displayed', true);
```

Comment:
you should use `toggleDisplayed` since it's the public API to manipulate `displayed`. Then you have to assert both the `displayed` value and the class on the container.